### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by marcionicolau** [rgbkrk/atom-script#295]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:54:11)
ScriptView.initialize (/Users/user/.atom/packages/script/lib/script-view.coffee:26:9)
```

**Comment by erran** [comment]
Fixed by #407, new release pending.

**Comment by erran** [comment]
Fixed by #407, new release pending.
